nion/data/xdata_1_0.py

```diff
@@ -91,8 +91,8 @@ def sum_region(data_and_metadata: _DataAndMetadataLike, mask_data_and_metadata:
 def average_region(data_and_metadata: _DataAndMetadataLike, mask_data_and_metadata: _DataAndMetadataLike) -> DataAndMetadata.DataAndMetadata:
     return Core.function_average_region(data_and_metadata, mask_data_and_metadata)
 
-def rebin_image(data_and_metadata: _DataAndMetadataLike, shape: DataAndMetadata.ShapeType) -> DataAndMetadata.DataAndMetadata:
```

Comment:
This is redefining the behavior of an existing function - which I don't think we should do.

Can you leave the old function for backwards compatibility and then rename the new function to something like `rebin_factor`?

Here are some references in support of this name, but I'm not set on the new name:
https://scipy-cookbook.readthedocs.io/items/Rebinning.html
https://www.l3harrisgeospatial.com/docs/rebin.html

There's also this if you want to go this route:
https://www.mathworks.com/help/matlab/ref/double.discretize.html#d117e325778